### PR TITLE
docs: add ground segmentation

### PR DESCRIPTION
**Pull request by nanoshimarobot**
## Types of PR

- [x] Upgrade of existing features

## Description
Add document for ground segmentation evaluator

### CODE REVIEW

File: docs/use_case/ground_segmentation.en.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Evaluate Ground Segmentation
+
+Evaluate the function of segmenting and removing ground point clouds from input point clouds.
```
Comment (badai-nguyen) [inline docs/use_case/ground_segmentation.en.md:3]:
```suggestion
Evaluate the performance of the Obstacle Segmentation sub-component in Autoware, which is responsible for identifying point clouds originating from obstacle that vehicle should avoid.
```

File: docs/use_case/ground_segmentation.en.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+## Ground Truth data
+
+The Ground Truth data required for evaluation can be provided using the following two methods, and each can be used by changing the `Evaluation.Conditions.Method` of the scenario.
+
+### annotated_rosbag
+
+This method involves adding a field to the point cloud data in the bag file to represent semantic labels.
+
+Synchronize and subscribe to topics before and after ground removal, and evaluate accuracy by comparing the number of points with ground labels.
```
Comment (badai-nguyen) [inline docs/use_case/ground_segmentation.en.md:13]:
```suggestion
Synchronize and subscribe to topics before and after ground removal, and evaluate the accuracy by comparing the number of points with ground and non-ground labels.
```

File: docs/use_case/ground_segmentation.en.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+In this evaluation framework, the semantic labels are assumed to be recorded in an `INT32` field named `entity_id`.
+
+### annotated_pcd
+
+This method involves adding a field to the point cloud data provided as a dataset (~/driving_log_replayer_v2/ground_segmentation/dataset/data/LIDAR_CONCAT/\*.pnd.bin) to represent semantic labels.
```
Comment (badai-nguyen) [inline docs/use_case/ground_segmentation.en.md:19]:
```suggestion
This method involves adding a field to the point cloud data provided as a dataset (~/driving_log_replayer_v2/ground_segmentation/dataset/data/LIDAR_CONCAT/\*.pcd.bin) to represent semantic labels.
```

File: docs/use_case/ground_segmentation.en.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+| topic name | Data type |
+| ---------- | --------- |
+| -          | -         |
+
+## Arguments passed to logging_simulator.launch
+
+- localization: false
+- planning: false
+- control: false
```
Comment (badai-nguyen) [inline docs/use_case/ground_segmentation.en.md:88]:
```suggestion
- control: false
- sensing: false
```

**Review by badai-nguyen — COMMENTED** [review]
@nanoshimarobot Thank you for your PR. I have some minor suggestions as below.

File: docs/use_case/ground_segmentation.ja.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+bagデータに含まれる点群データに、セマンティックラベルを表すフィールドを持たせる方法。
+
+地面点群除去前後の topic を同期 subscribe し、地面ラベルを持つ点数の比較により精度評価を行う。
+
+本評価基盤では、セマンティックラベルは`INT32`型の`entity_id`フィールドに記述されていることが前提となっている。
+
+### annotated_pcd
+
+データセットとして与える点群データ(`~/driving_log_replayer_v2/ground_segmentation/dataset/data/LIDAR_CONCAT/*.pnd.bin`)に、セマンティックラベルを表すフィールドを持たせる方法。
```
Comment (badai-nguyen) [inline docs/use_case/ground_segmentation.ja.md:19]:
```suggestion
データセットとして与える点群データ(`~/driving_log_replayer_v2/ground_segmentation/dataset/data/LIDAR_CONCAT/*.pcd.bin`)に、セマンティックラベルを表すフィールドを持たせる方法。
```

File: docs/use_case/ground_segmentation.ja.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+| topic 名 | データ型 |
+| -------- | -------- |
+| -        | -        |
+
+## logging_simulator.launch に渡す引数
+
+- localization: false
+- planning: false
+- control: false
```
Comment (badai-nguyen) [inline docs/use_case/ground_segmentation.ja.md:87]:
```suggestion
- control: false
- sensing: false
```

**Review by badai-nguyen — DISMISSED** [review]
LGTM

File: docs/use_case/ground_segmentation.ja.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+
+bagデータに含まれる点群データに、セマンティックラベルを表すフィールドを持たせる方法。
+
+地面点群除去前後の topic を同期 subscribe し、地面ラベルを持つ点数の比較により精度評価を行う。
+
+本評価基盤では、セマンティックラベルは`INT32`型の`entity_id`フィールドに記述されていることが前提となっている。
+
+### annotated_pcd
+
+データセットとして与える点群データ(`~/driving_log_replayer_v2/ground_segmentation/dataset/data/LIDAR_CONCAT/*.pcd.bin`)に、セマンティックラベルを表すフィールドを持たせる方法。
```
Comment (hayato-m126) [inline docs/use_case/ground_segmentation.ja.md:19]: 点群データ(`~/driving_log_replayer_v2/ground_segmentation/dataset/data/LIDAR_CONCAT/*.pcd.bin`)
と書いてしまうと、$HOMEのこのディレクトリしか受け付けられないように見えるので、

点群データ(`dataset/data/LIDAR_CONCAT/*.pcd.bin`)ですかね。

File: driving_log_replayer_v2/driving_log_replayer_v2/launch/ground_segmentation.py [hunk 8]
```diff
@@ -20,6 +20,7 @@
 """
 
 AUTOWARE_DISABLE = {
+    "sensing": "false",
```
Comment (hayato-m126) [inline driving_log_replayer_v2/driving_log_replayer_v2/launch/ground_segmentation.py:23]: @badai-nguyen 
I think `sensing:false` is necessary based on the launch argument section in the documentation.

File: docs/use_case/ground_segmentation.ja.md (hunk 7 above)
Comment (hayato-m126) [reply docs/use_case/ground_segmentation.ja.md]: I fixed

**Pull request merged**